swiftide/src/transformers/metadata_qa_code.rs

```diff
@@ -91,12 +91,16 @@ impl Transformer for MetadataQACode {
     /// This function will return an error if the `SimplePrompt` client fails to generate a response.
     #[tracing::instrument(skip_all, name = "transformers.metadata_qa_code")]
     async fn transform_node(&self, mut node: Node) -> Result<Node> {
-        let prompt = self
+        let mut prompt = self
             .prompt_template
             .to_prompt()
             .with_node(&node)
             .with_context_value("questions", self.num_questions);
 
+        if let Some(context) = node.metadata.get("Context (code)") {
+            prompt = prompt.with_context_value("context", context.clone());
```

Comment:
You can probably pass the context as ref too, no need for the extra allocation.

Reply:
Ah nice I was wondering about that.